zerver/webhooks/heroku/view.py

```diff
@@ -14,7 +14,7 @@ def api_heroku_webhook(request: HttpRequest, user_profile: UserProfile,
                        head: str=REQ(), app: str=REQ(), user: str=REQ(),
                        url: str=REQ(), git_log: str=REQ()) -> HttpResponse:
     template = "{} deployed version {} of [{}]({})\n> {}"
-    content = template.format(user, head, app, url, git_log)
+    content = template.format(user, head, app, url, git_log.replace('\n', '\n> '))
```

Comment:
I'd rather use Zulip's block-quote format for this, rather than doing a bunch of `>`s.  E.g.
~~~
``` quote
git_log
```
~~~

Reply:
@timabbott Thanks for the review! I pushed this change.